web/packages/core/src/public/v1.ts

```diff
@@ -0,0 +1,173 @@
+import { MovieMetadata } from "../movie-metadata";
+import { DataLoadOptions, URLLoadOptions } from "../load-options";
+import { ReadyState } from "../internal/player/inner";
+
+export interface PlayerV1 {
+    /**
+     * A movie can communicate with the hosting page using fscommand
+     * as long as script access is allowed.
+     *
+     * @param command A string passed to the host application for any use.
+     * @param args A string passed to the host application for any use.
+     * @returns True if the command was handled.
+     */
+    onFSCommand: ((command: string, args: string) => boolean) | null;
```

Comment:
3 possibilities here:

As today, we set the callback for all FS commands.
We change it to register a callback for all FS commands.
We change it to register the callback for a specific named FS command.

I think I kinda prefer 2.

Reply:
We could also consider switching the player to be a proper `EventTarget` and use `addEventListener` for this.

Reply:
If we go `EventTarget` we'll lose the warning of unhandled commands, unless we require people to flag that they've handled it. Right now we ask that the `onFSCommand` returns true when handled.

I think that's probably fine, but what do you think?

Reply:
I thought about this a bit and decided that event may not be the way to go. A few issues:
1. Versioning will be difficult
2. It's a bit weird to have the element have the event when everything else is behind `ruffle()` - people may accidentally `ruffle().addEventListener`
3. Typescript really does not seem to like custom events... makes it super verbose to use :(

I'll go with option 2 for now and we can reevaluate in the future for a V2.